Error in building MATLAB wrapper:   TypeError: unhashable type: 'Typename'

I am trying to build the MATLAB wrapper. Getting the following error

...
[ 94%] Built target SmartProjectionFactorExample
[ 94%] Built target FixedLagSmootherExample
[ 97%] Built target SmartRangeExample_plaza1
[ 97%] Generating ../wrap/gtsam/gtsam_wrapper.cpp
[MatlabWrapper] Ignoring classes: ['gtsam::Point2', 'gtsam::Point3', 'gtsam::CustomFactor']
Traceback (most recent call last):
  File "/home/air/gtsam-develop/wrap/scripts/matlab_wrap.py", line 58, in <module>
    cc_content = wrapper.wrap(sources, path=args.out)
  File "/home/air/gtsam-develop/wrap/gtwrap/matlab_wrapper/wrapper.py", line 1639, in wrap
    self.wrap_namespace(module)
  File "/home/air/gtsam-develop/wrap/gtwrap/matlab_wrapper/wrapper.py", line 1034, in wrap_namespace
    self.wrap_namespace(element)
  File "/home/air/gtsam-develop/wrap/gtwrap/matlab_wrapper/wrapper.py", line 1040, in wrap_namespace
    class_text = self.wrap_instantiated_class(
  File "/home/air/gtsam-develop/wrap/gtwrap/matlab_wrapper/wrapper.py", line 964, in wrap_instantiated_class
    self.wrap_class_constructors(
  File "/home/air/gtsam-develop/wrap/gtwrap/matlab_wrapper/wrapper.py", line 660, in wrap_class_constructors
    varargin=self._wrap_variable_arguments(
  File "/home/air/gtsam-develop/wrap/gtwrap/matlab_wrapper/wrapper.py", line 237, in _wrap_variable_arguments
    check_type = self._format_type_name(
  File "/home/air/gtsam-develop/wrap/gtwrap/matlab_wrapper/mixins.py", line 134, in _format_type_name
    self._format_type_name(type_name.instantiations[idx],
  File "/home/air/gtsam-develop/wrap/gtwrap/matlab_wrapper/mixins.py", line 112, in _format_type_name
    formatted_type_name += self.data_type.get(name) or name
TypeError: unhashable type: 'Typename'
make[2]: *** [matlab/CMakeFiles/gtsam_matlab_wrapper.dir/build.make:76：wrap/gtsam/gtsam_wrapper.cpp] error 1
make[1]: *** [CMakeFiles/Makefile2:32161：matlab/CMakeFiles/gtsam_matlab_wrapper.dir/all] error 2
make: *** [Makefile:163：all] error 2

Could you help me to find what's wrong with it?
Python 3.8.10
Ubuntu 20.04.1 LTS
GTSAM 4.1.1

--- a/gtsam/geometry/geometry.i
+++ b/gtsam/geometry/geometry.i
@@ -873,7 +873,7 @@ template <CALIBRATION>
 class PinholeCamera {
   // Standard Constructors and Named Constructors
   PinholeCamera();
-  PinholeCamera(const gtsam::PinholeCamera<CALIBRATION> other);
+  PinholeCamera(const This other);
   PinholeCamera(const gtsam::Pose3& pose);
   PinholeCamera(const gtsam::Pose3& pose, const CALIBRATION& K);
   static This Level(const CALIBRATION& K, const gtsam::Pose2& pose,
@@ -942,7 +942,7 @@ template <CALIBRATION>
 class PinholePose {
   // Standard Constructors and Named Constructors
   PinholePose();
-  PinholePose(const gtsam::PinholePose<CALIBRATION> other);
+  PinholePose(const This other);
   PinholePose(const gtsam::Pose3& pose);
   PinholePose(const gtsam::Pose3& pose, const CALIBRATION* K);
   static This Level(const gtsam::Pose2& pose, double height);

